jupyterbook/content/code_gallery/data_management_notebooks/2023-03-20-Reading_and_writing_zarr.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "29370978",
+   "metadata": {},
+   "source": [
+    "# Reading and writting zarr files with xarray\n",
+    "\n",
+    "\n",
+    "Created: 2023-03-20\n",
+    "\n",
+    "\n",
+    "\n",
+    "[The zarr format](https://zarr.readthedocs.io/en/stable/) is a file storage based specification for chunked, compressed, N-dimensional arrays. The formart is based on an open-source specification and its main goal is to make cloud data read/write a bit easier and more effective.\n",
+    "\n",
+    "The main propblems in data storage are:\n",
+    "\n",
+    "1. Read/write data that is larger than memory\n",
+    "1. Being able to parallelize computations\n",
+    "1. Reduce the I/O botteneck\n",
+    "1. Compression\n",
+    "1. Speed\n",
+    "\n",
+    "One solution is to use a chunked parallel computing framework and a chnked parallel storage library. Zarr helps us with the latter.\n",
```

Comment:
It might be worth explaining what "chunked" means in this context. How is it different than compressed?

Reply:
Added in the last commit. Let me know if that is OK.